conftest.py

```diff
@@ -37,3 +54,59 @@ def _open_browser(url):
 
             webbrowser.open_new_tab(url)
     return _open_browser
+
+
+@pytest.fixture
+async def compare_screenshots(request):
+    """Run visual regression test on the output."""
+    async def _compare_screenshots(test_id, url):
+        if (request.config.getoption(COMPARE_SCREENSHOTS_OPTION) or
+            request.config.getoption(UPDATE_REFERENCE_SCREENSHOTS_OPTION)):
+            from playwright.async_api import async_playwright
+
+            # Filtering warnings generated by playwright
+            warnings.filterwarnings(
+                'ignore', category=DeprecationWarning, module='pyee.*')
+
+            test_dir = Path(__file__).parent / 'docrepr' / 'tests'
```

Comment:
Not great that we have to hardcode this path here, but you put the fixture in a `conftest` in the test directory or directly in `test_output`, you'd have to hardcod the option names instead, so I guess this is the least bad approach. Maybe at least factor this into a top-level constant, `TEST_DIR`, so if it does change, we can change it one obvious place?